packages/runtime/plugin-runtime/src/router/cli/code/index.ts

```diff
@@ -17,6 +19,7 @@ import type {
   NestedRouteForCli,
 } from '@modern-js/types';
 import { AppNormalizedConfig, AppTools } from '@modern-js/app-tools';
+import { cloneDeep } from '@modern-js/utils/lodash';
```

Comment:
The plugin-runtime will import lodash, the package size will increase to much.
Can we use other way to replace?

Reply:
It is a cli plugin, not included in the client bundle

Reply:
Got it ,my mistake